src/containers/FileExplorer/FilePreview.tsx

```diff
@@ -193,13 +192,20 @@ class MapModelToAssetForm extends React.Component<Props, State> {
   }
 
   fetchFileUrl = async () => {
-    const { selectedDocument } = this.props;
-    const [url] = await sdk.files.getDownloadUrls([
-      { id: selectedDocument.id },
-    ]);
-    this.setState({
-      filePreviewUrl: url.downloadUrl,
-    });
+    this.setState(
+      {
+        filePreviewUrl: undefined,
```

Comment:
This cancels the PDF viewer so it can reload since the Viewer is too dumb to reload URL other wise.